Source/Scene/Primitive.js

```diff
@@ -545,7 +545,11 @@ define([
         var indices;
         if (defined(geometry.indices)) {
             var sourceValues = geometry.indices;
-            indices = new sourceValues.constructor(sourceValues);
+            if (isArray(sourceValues)) {
```

Comment:
I thought a change like this went into master recently.  If so, can we sync up; if not, cherry-pick and open a PR into master.